pkg/proxy/parser.go

```diff
@@ -767,6 +794,10 @@ func isLinux(p *model.Platform) bool {
 	return isPlatform(p, linux)
 }
 
+func isWindows(p *model.Platform) bool {
+	return isPlatform(p, windows)
+}
+
 func isPlatform(p *model.Platform, platform string) bool {
 	name := strings.ToLower(p.Name)
 	os := strings.ToLower(p.BaseOs)
```

Comment:
There don't seem to be any changes in current codes that need checking, fixing, testing or suggesting improvements. The code looks well-established with minimal differences from the previous release of 2021-09-01 which suggests it hasn't been extensively tested against new environments and APIs yet. If there are specific areas you want checked or reviewed such as API implementation, database connections, performance etc then let us know!